webpack.config.js

```diff
@@ -5,30 +5,26 @@ const pkg = require('./package.json')
 module.exports = {
   mode: 'production',
   devtool: 'source-map',
-  entry: './src/standalone.js',
+  entry: './src/standalone.tsx',
   output: {
     path: path.join(__dirname, 'standalone'),
     filename: 'consent-manager.js',
     library: 'consentManager'
   },
   resolve: {
     alias: {
-      react: 'inferno-compat',
-      'react-dom': 'inferno-compat',
+      react: 'preact-compat',
+      'react-dom': 'preact-compat',
```

Comment:
Just FYI that I ran into a bunch of random edge issues when trying to use Preact, which is why I went with Inferno instead.

Reply:
I remember that. Things look fairly safe so far (based on my manual testing), but I'll keep QAing everything before releasing something new.